src/main/java/org/jabref/gui/autocompleter/SuggestionProviders.java

```diff
@@ -22,14 +22,26 @@ public SuggestionProviders(BibDatabase database, JournalAbbreviationRepository a
         this.isEmpty = false;
     }
 
+    public SuggestionProviders(BibDatabase database) {
+        this.database = database;
+        this.isEmpty = true;
+    }
+
     public SuggestionProviders() {
         this.isEmpty = true;
     }
 
     public SuggestionProvider<?> getForField(Field field) {
         if (isEmpty || !autoCompletePreferences.getCompleteFields().contains(field)) {
+            Set<FieldProperty> fieldProperties = field.getProperties();
+            if (fieldProperties.contains(FieldProperty.SINGLE_ENTRY_LINK)) {
+                return new BibEntrySuggestionProvider(database);
+            }
             return new EmptySuggestionProvider();
         }
+//        if (isEmpty || !autoCompletePreferences.getCompleteFields().contains(field)) {
```

Comment:
Remove commented out code, that's why we have git, you can always go back in the history and restore deleted code.

Reply:
Removed those in new commit